src/lib/icons/access-code-key.ts

```diff
@@ -0,0 +1,4 @@
+export const accessCodeKey = `<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We should make the bundler hanlde svg => react component. Let's add that to the discussion: https://github.com/seamapi/react/discussions/36